### DISCUSSION TITLE
eNyugta másolat 2.

### DISCUSSION THREAD

**Discussion by MicraMetripond** [nav-gov-hu/eRECEIPT#17] [Kérdések és válaszok]
Mikor, meddig és hányszor lehet egy nyugtáról papíralapú másolatot kiállatni?

**Comment by renced42** [comment]
Kedves @MicraMetripond 

Jelenleg alapvetően másolatot a vásárláskor lehet kiállítani a vevő kérésére fizetést követően. A pénztártól való távozást követően ennek már nincs jelentősége mert vagy van a vevőnek másolata vagy van QR kódja vagy vevői appja van amiben a nyugta letöltődött.
Mi alapján kérhet valaki másolatot utólag?
